content/en/containers/cluster_agent/_index.md

```diff
@@ -55,10 +55,14 @@ Some features related to later Kubernetes versions require a minimum Datadog Age
     {{< nextlink href="/agent/cluster_agent/troubleshooting" >}}<u>Cluster Agent Troubleshooting</u>: Find troubleshooting information for the Datadog Cluster Agent.{{< /nextlink >}}
 {{< /whatsnext >}}
 
+## Monitoring the Cluster Agent
+The Datadog Agent includes an integration that automatically monitors the Cluster Agent. This runs on the regular Datadog Agent pod that is on the same node as the Cluster Agent. It will not run in the Cluster Agent itself. Refer to the [Datadog Cluster Agent integration documentation][4] for details.
```

Comment:
```suggestion
The Datadog Agent includes an integration that automatically monitors the Cluster Agent. The integration runs on the regular Datadog Agent pod that is on the same node as the Cluster Agent. It will not run in the Cluster Agent itself. Refer to the [Datadog Cluster Agent integration documentation][4] for details.
```